pkgs/applications/misc/appeditor/default.nix

```diff
@@ -15,7 +15,7 @@
 
 stdenv.mkDerivation rec {
   pname = "appeditor";
-  version = "1.1.0";
+  version = "1.1.1";
 
   src = fetchFromGitHub {
     owner = "donadigo";
```

Comment:
Why is the hash not changed?

Reply:
It's trusting the updateScript to work correctly.